docs/layout_styles.rst

```diff
@@ -572,8 +572,9 @@ Own CSS file on sphinx level
 If you want to use most of the sphinx-needs internal styles but only need some specific changes for single elements, you
 can provide your own CSS file by register it inside your conf.py::
 
-    def setup(app):
-        app.add_stylesheet('css/my_custom.css')  # may also be an URL
+    html_css_files = ['css/my_custom.css'])  # may also be an tuple
```

Comment:
")" at the end not needed.

"# may also be an tuple": I would remove this part.
Was hard for me to understand, what benefits a tuple has and why it is needed.
The later link to the docs makes it clear. But first it was confusing and this feature may not be needed for most users.